Formula/burp.rb

```diff
@@ -2,11 +2,10 @@ class Burp < Formula
   desc "Network backup and restore"
   homepage "https://burp.grke.org/"
   license "AGPL-3.0"
-  revision 1
 
   stable do
-    url "https://downloads.sourceforge.net/project/burp/burp-2.2.18/burp-2.2.18.tar.bz2"
-    sha256 "9c0c5298d8c2995d30d4e1a63d2882662e7056ce2b0cee1f65d7d0a6775c0f81"
+    url "https://github.com/grke/burp/releases/download/2.4.0/burp-2.4.0.tar.bz2"
```

Comment:
livecheck also needs to be updated.

Reply:
I had updated to current stable release, which could be seen in https://burp.grke.org/download.html. This would require parsing out text like with Stable text.

Current latest version is 3.1.0 which shows up via git livecheck. It looks like we have been tracking stable releases (just that current stable version isn't uploaded to sourceforge).